lib/active_record/acts_as/querying.rb

```diff
@@ -28,10 +28,9 @@ def where!(opts, *rest)
 
     module ScopeForCreate
       def scope_for_create(attributes = nil)
-        scope = ActiveRecord.version.to_s.to_f >= 5.2 ? super(attributes) : where_values_hash
-        if acting_as?
-          scope.merge!(where_values_hash(acting_as_model.table_name))
-        end
+        scope = where_values_hash
+        scope = values_for_create(attributes) if respond_to?(:values_for_create)
```

Comment:
Could you combine these lines to `scope = respond_to?(:values_for_create) ? values_for_create(attributes) : where_values_hash`?

Reply:
Are your sure, @manuelmeurer? According to `rubocop`, the code below is against its guidelines:

```
scope = respond_to?(:values_for_create) ? values_for_create(attributes) : where_values_hash
```

Reply:
The let's use a standard `if else`. :)
Assigning `scope` and then reassigning it on the next line is not good style IMO.

Reply:
@manuelmeurer, okay sure 😄 I will push changes in a moment.

Reply:
@manuelmeurer, I've made the changes, see commit https://github.com/krautcomputing/active_record-acts_as/pull/29/commits/645364b15e388442f764b5e35a54d9e1e61b625f. Thanks.